pkg/ovs/openflow/ofctrl_builder.go

```diff
@@ -302,31 +303,35 @@ func ctLabelRange(high, low uint64, rng *Range, match *ofctrl.FlowMatch) {
 	match.CtLabelLo = low
 	match.CtLabelHiMask = 0xffff_ffff_ffff_ffff
 	match.CtLabelLoMask = 0xffff_ffff_ffff_ffff
-	if rng[0] == rng[1] {
-		if rng[0] < 64 {
-			match.CtLabelLoMask = 1 << rng[0]
-			match.CtLabelHiMask = 0
-		} else {
-			match.CtLabelHiMask = 1 << (rng[0] - 64)
-			match.CtLabelLoMask = 0
-		}
-	} else if rng[0] < 64 && rng[1] >= 64 {
-		match.CtLabelLoMask <<= rng[0]
-		match.CtLabelHiMask >>= 127 - rng[1]
-	} else if rng[1] < 64 {
+	if rng[1] < 64 {
 		match.CtLabelLoMask &= 0xffff_ffff_ffff_ffff << rng[0]
 		match.CtLabelLoMask &= 0xffff_ffff_ffff_ffff >> (63 - rng[1])
+		match.CtLabelLo &= match.CtLabelLoMask
```

Comment:
I thought in Antrea code, CTLabel will be always used with a mask, this re-calculate can fix the wrong value with mask, for example, 0xff/0xf, it is incorrect value for mask 0xf, and it can be fixed to 0xf/0xf by this re-calculate.

Reply:
Firstly, OVS should perform the mask operation as your example shows. Secondly, if the value is set with 0xf/0xf0, after this mask operation OVS should receive with 0x0/0xf, I don't think this is what the caller expected. In stead, I think we should check if the value is shifted correctly, either in the caller, or in this package. In my mind, ofnet does not shift the data according to the mask.

Reply:
Ok, I'll change the re-calculate, but I don't got that `check if the value is shifted correctly`.